### PR TITLE
TS-6952 update for otp22

### PR DESCRIPTION
**Pull request by TC-aLi**

### CODE REVIEW

**Comment by TC-aLi** [comment]
There is a Travis issue which blocks rebar3 from getting deps. Nikhil and Travis team are on it. All test cases pass on my computer. Merge now.

**Review by TT-lwang — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,8 @@
 *~
 ebin/
 ebin/*.beam
-/deps
\ No newline at end of file
+/deps
```
Comment (anderswei) [inline .gitignore:8]: remove the leading /

File: Makefile [hunk 2]
```diff
@@ -1,15 +1,19 @@
 all:
-	./rebar get-deps compile
+	rebar3 get-deps compile
```
Comment (anderswei) [inline Makefile:2]:
```suggestion
rebar3 get-deps compile
```

File: .gitignore (hunk 1 above)
Comment (anderswei) [reply .gitignore:8]:
```suggestion
deps
```

**Pull request merged**